src/teaching-element/Assessment/MultipleChoice.vue

```diff
@@ -45,16 +46,21 @@ export default {
     }
   },
   methods: {
-    update() {
-      let { userAnswer } = this;
-      userAnswer = isEmpty(userAnswer) ? null : userAnswer.sort();
-      this.$emit('update', { userAnswer });
+    getAnswerClass(answerIndex) {
+      if (!this.disabled || !this.options.showCorrectChoices) return null;
```

Comment:
Let's rename this `options` prop to `setCorrectnessClass` and also add it to all other assessment components so weather classes will be set can be controlled from the outside for any type of assessment. 
Also, let's default it to true.